examples/gno.land/r/demo/pagination/pagination.gno

```diff
@@ -0,0 +1,137 @@
+// This realm demonstrates a pagination system working with gnoweb
+// It keeps a slice of signed up addresses for efficient pagination
+package pagination
+
+import (
+	"std"
+	"strconv"
+
+	"gno.land/p/demo/avl"
+	"gno.land/p/demo/mux"
+	"gno.land/p/demo/ufmt"
+)
+
+type Signup struct {
+	account string
+	height  int64
+}
+
+// signups - keep a slice of signed up addresses efficient pagination
+var signups []Signup
+
+// tracker - keep track of who signed up
+var tracker *avl.Tree
+
+const pageSize = 20
+
+func init() {
+	tracker = avl.NewTree()
+	SignUp()
+}
+
+func SignUp() string {
+	caller := std.GetOrigCaller().String()
+	height := std.GetHeight()
+
+	_, exists := tracker.Get(caller)
```

Comment:
```suggestion
	if _, exists := tracker.Get(caller); exists {
```

Reply:
And then remove the next line.

Reply:
[152fe8b](https://github.com/gnolang/gno/pull/1949/commits/152fe8b3daf7b62673800b0f73d19f62b063b665)